### ISSUE TITLE
Usability ( for screen readers) - title missing from analysis iframe

### ISSUE THREAD

**Issue by Wildebrew** [dequelabs/axe-firefox-devtools#26]
The iFrame containing the analysis results does not have a title
attribute. Screen readers recognize it as data-axe-panel. While it is not a violation of WCAG 2.4.1 
to have an iframe without a title, it does help screen reader users when a descriptive title is provided. Add a title such as "AXe Analysis results" or similar.
